src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -72,5 +96,9 @@ public ListBoxModel doFillMergeStrategyItems() {
                 m.add(strategy.toString(), strategy.toString());
             return m;
         }
+
+        public MergeCommand.GitPluginFastForwardMode getFastForwardMode() {
```

Comment:
This shouldn't be in Descriptor. Descriptor is global singleton configuration. 
doFillMergeStrategyItems() was created here only for providing available entries in select form.